src/index.js

```diff
@@ -4,38 +4,39 @@ import CGUs from './app/index.js';
 import * as logger from './logger/index.js';
 import Notifier from './notifier/index.js';
 
+const serviceId = process.argv.slice(process.argv.indexOf('--serviceId'))[1];
```

Comment:
See the way this could be implemented in #122 (c8437bbf9ba77226f6582540e01c81414f1a923a). In any case, unify with arguments parsing code in #110 or #122 🙂